### PR TITLE
Edit Movies Completed Before Peer Check

### PR DESCRIPTION
**Pull request by SCapel2**

### CODE REVIEW

**Review by Jacopo-Magnani — APPROVED** [review]
Looks good

File: templates/edit_movies_form.html [hunk 1]
```diff
@@ -9,15 +9,15 @@ <h1 class="mb-5">Edit Movie</h1>
 <form method="post">
```
Comment (abbategabriel) [inline templates/edit_movies_form.html:9]: I would add action="/movies/{{movie_id}}" after method="post". I tested it in my vs code and it didnt work without it.

**Review by abbategabriel — CHANGES_REQUESTED** [review]
I would change the edit_movies_page function to be like this:
@app.get('/movies/<int:movie_id>/edit')
def get_edit_movies_page(movie_id: int):
    return render_template('edit_movies_form.html', movie_id = movie_id)
That way when you write the form in html and add action="/movies/{{movie_id}}" it works.
I'm not 100% sure but thats what I did for my form and + when i edited this in vs code it made it work because I tried running it before and it was saying method not allowed when I would submit the form.

**Comment by mlynnlato** [comment]
> I would change the edit_movies_page function to be like this: @app.get('/movies/int:movie_id/edit') def get_edit_movies_page(movie_id: int): return render_template('edit_movies_form.html', movie_id = movie_id) That way when you write the form in html and add action="/movies/{{movie_id}}" it works. I'm not 100% sure but thats what I did for my form and + when i edited this in vs code it made it work because I tried running it before and it was saying method not allowed when I would submit the form.

Going along with this, 
`@app.get('/movies/<int:movie_id>/edit')
def get_edit_movies_page(movie_id: int):
    return render_template('edit_movies_form.html', movie_id = movie_id)`
and adding  `<form method="post" action="/movies/{{movie_id}}" after method="post">` to _edit_movies_form.html _ allowed for app to function as expected.

File: templates/edit_movies_form.html (hunk 1 above)
Comment (SCapel2) [reply templates/edit_movies_form.html]: Thank you, I fixed it

**Pull request merged**